interface/ApfpBlas.cpp

```diff
@@ -0,0 +1,195 @@
+#include "ApfpBlas.h"
+#include "Apfp.h"
+#include "Config.h"
+#include <optional>
+#include <stdexcept>
+
+static std::optional<Apfp> apfp;
+static std::string last_error_message;
+
+enum ApfpBlasUplo : char {
+    upper = 'U',
+    lower = 'L'
+};
+
+enum ApfpBlasTrans : char {
+    normal = 'N',
+    transpose = 'T',
+};
+
+int ApfpInit(unsigned long precision) {
```

Comment:
Instead of all these `Apfp` prefixes, why not stick all of this in an `apfp` namespace? This looks a bit too much like C code ;-)